src/main/groovy/net/neoforged/gradleutils/tasks/ExtractActionsWorkflowsTask.java

```diff
@@ -57,6 +73,10 @@ protected void extractZip(final String fileZip, final File destDir) throws Excep
 
                 FileOutputStream fos = new FileOutputStream(newFile);
                 // write file content
+                fos.write(String.format(
+                        "# File generated by the GradleUtils `%s` task, avoid modifying it directly\n# The template can be found at https://github.com/neoforged/GradleUtils/blob/%s/src/actionsTemplate/resources/%s\n\n", getName(),
+                        gitCommit, zipEntry.getName()
```

Comment:
Put the `getName()` call on the next line, with the other arguments.
```suggestion
                        "# File generated by the GradleUtils `%s` task, avoid modifying it directly\n# The template can be found at https://github.com/neoforged/GradleUtils/blob/%s/src/actionsTemplate/resources/%s\n\n", 
                        getName(), gitCommit, zipEntry.getName()
```